dispatch/dispatch.h

```diff
@@ -49,7 +49,7 @@
 
 #if defined(__linux__) && defined(__has_feature)
 #if __has_feature(modules)
-#include <stdio.h> // for off_t (to match Glibc.modulemap)
+//#include <stdio.h> // for off_t (to match Glibc.modulemap)
```

Comment:
commenting out is not the right patch anyway, this is an API header, you just don't do that. either the full hunk is to be removed, or it should stay. @dgrove-oss or @seabaylea should advise here.